package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openapi-format",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
Can you reset this?
Since this will be done by the release process.

Reply:
done